rails/Gemfile

```diff
@@ -87,6 +87,8 @@ group :development do
 
   # annotate models
   gem "annotate"
+
+  gem "foreman"
```

Comment:
Per `foreman`'s installation instructions, this gem [should NOT be installed in the Gemfile](https://github.com/ddollar/foreman#installation).